spec/index.bs

```diff
@@ -176,7 +178,7 @@ spec: WebIDL; urlPrefix: https://heycam.github.io/webidl/#
         Let <var>currentRegistration</var> be the <a lt="sync registration">registration</a> in <var>serviceWorkerRegistration</var>'s <a>list of sync registrations</a> whose <a>tag</a> equals <var>options.tag</var> if it exists, else null.
       </li>
       <li>
-        If <var>currentRegistration</var> is not null, <a>resolve</a> <var>promise</var> with a new {{SyncRegistration>> instance associated with <var>currentRegistration</var> and abort these steps.
+        If <var>currentRegistration</var> is not null, <a>resolve</a> <var>promise</var> with a new {{SyncRegistration}} instance associated with <var>currentRegistration</var> and abort these steps.
```

Comment:
Ah yes, I wasn't sure about this case, but it makes sense that register with an existing tag should also potentially trigger a sync event immediately. Added text for that.